messages/web.login.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "authorizes a Salesforce org by opening a browser so you can log in through salesforce.com.",
+  "examples": [
+    "sfdx force:auth:web:login -a TestOrg1",
+    "sfdx force:auth:web:login -i <OAuth client id>",
+    "To log in to a sandbox, set --instanceurl to https://test.salesforce.com.",
+    "sfdx force:auth:web:login -r https://test.salesforce.com"
+  ],
+  "disableMasking": "disable masking of user input (for use with problematic terminals)",
+  "deviceWarning": "force:auth:web:login doesn't work when authorizing to a headless environment. Use force:auth:device:login instead.",
+  "clientSecretStdin": "OAuth client secret of personal connected app?",
```

Comment:
Please remove the ? symbol

Reply:
@rorolopetegui I'm porting these messages over from toolbelt without any modification: https://git.soma.salesforce.com/ALMSourceDrivenDev/force-com-toolbelt/blob/release/src/lib/messages.ts#L790